app/routes/_index/route.tsx

```diff
@@ -42,6 +42,7 @@ export default function HomePage() {
                                 name={product.name}
                                 price={product.priceData ?? undefined}
                                 className={styles.productCard}
+                                outOfStock={product.stock?.inventoryStatus === 'OUT_OF_STOCK'}
```

Comment:
there is defined enum for this in sdk. the only issue, find a correct name...

```
import {products as wixProducts  } from "@wix/stores"

...

product.stock?.inventoryStatus === wixProducts.InventoryStatus.IN_STOCK
```

Reply:
😮